examples/mnist_nonlinear.py

```diff
@@ -0,0 +1,77 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+from torchvision.datasets import MNIST
+
+# Note: this example requires the torchmetrics library: https://torchmetrics.readthedocs.io
+import torchmetrics
+from tqdm import tqdm
+
+import torchhd
+from torchhd import embeddings
+
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+print("Using {} device".format(device))
+
+DIMENSIONS = 10000
+IMG_SIZE = 28
+NUM_LEVELS = 1000
```

Comment:
This variable is not used anymore, could you remove it?

Reply:
Thanks for your quick review! I remove the `NUM_LEVELS` variable as you suggested. I also add one line of comment at the beginning of `mnist_nonlinear.py` example.